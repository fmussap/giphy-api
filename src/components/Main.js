import React from 'react'

import Header from './Header'
import Dashboard from './DashboardPage'

export const Main = () => (
  <div className='main'>
    <Header />
    <Dashboard />
  </div>
)

export default Main
