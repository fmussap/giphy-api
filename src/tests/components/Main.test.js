import React from 'react'
import { shallow } from 'enzyme'
import Main from '../../components/Main'

test('should render NotFoundPage correctly', () => {
  const wrapper = shallow(<Main />)
  expect(wrapper).toMatchSnapshot()
})
