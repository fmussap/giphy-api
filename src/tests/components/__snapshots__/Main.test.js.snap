// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFoundPage correctly 1`] = `
<div
  className="main"
>
  <Connect(Header) />
  <Connect(DashboardPage) />
</div>
`;
